source/offer-utilities.ts

```diff
@@ -76,23 +76,6 @@ function isOptionalProductToBePurchased(
   return selectableProductIds.has(product.selectableId);
 }
 
-/**
- * Extracts the ID of the entitlement product which is required to purchase a given product
- *
- * Currently originatingFromProductId is not present in DiscountRightSummary,
- * but will be implemented by ETU-21645.
- *
- * @see <a href="https://enturas.atlassian.net/browse/ETU-21645">ETU-21645 for status updates</a>
- */
-function extractOriginatingFromProductId(
```

Comment:
Because this is now a lot simpler, I just inlined it. Perhaps losing the explanation for the `originatingFromProductId` field's purpose makes our code less readable, but it's the Offers API documentation's job to explain this field's purpose. I'll take my chances.